pde2e-image/common/windows/common.ps1

```
@@ -6,9 +6,9 @@ function Command-Exists($command) {
     return $?
 }
 
-# Download Podman Desktop
-function Download-PD($fileName) {
-    Write-Host "Downloading Podman Desktop from $pdUrl and saving to $fileName"
+# Download application binary
+function Download-App($fileName) {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Download-App' uses an unapproved verb.

(PSUseApprovedVerbs)